packages/prisma/seed.ts

```diff
@@ -911,6 +924,7 @@ async function main() {
       name: "Platform Team",
       slug: "platform-admin-team",
       isPlatform: true,
+      isOrganization: true,
```

Comment:
Previously platform billing check was failing because there's a `OrganizationRolesGuard` which checks for the `isOrganization` to be true, and since that was not set to true the billing check would always fail.